core/domain/stats_domain.py

```diff
@@ -537,16 +560,20 @@ def __init__(self, answer, frequency):
         self.answer = answer
         self.frequency = frequency
 
+    def to_raw_pair(self):
```

Comment:
I'd avoid introducing this. I don't think it's fundamental to AnswerOccurrence; do it in the caller instead. Also, "raw" is the name is misleading as the answer is not converted.